demos/imagenet_classification_demo/main.cpp

```diff
@@ -0,0 +1,357 @@
+                }
+
+                // for CPU execution, more throughput-oriented execution via streams
+                //if (FLAGS_api == "async")
+                ie.SetConfig({{ CONFIG_KEY(CPU_THROUGHPUT_STREAMS),
+                                (device_nstreams.count(device) > 0 ? std::to_string(device_nstreams.at(device)) :
+                                                                                 "CPU_THROUGHPUT_AUTO") }}, device);
+                device_nstreams[device] = std::stoi(ie.GetConfig(device, CONFIG_KEY(CPU_THROUGHPUT_STREAMS)).as<std::string>());
+            } else if (device == ("GPU")) {
+                //if (FLAGS_api == "async")
+                ie.SetConfig({{ CONFIG_KEY(GPU_THROUGHPUT_STREAMS),
+                                (device_nstreams.count(device) > 0 ? std::to_string(device_nstreams.at(device)) :
+                                                                         "GPU_THROUGHPUT_AUTO") }}, device);
+                device_nstreams[device] = std::stoi(ie.GetConfig(device, CONFIG_KEY(GPU_THROUGHPUT_STREAMS)).as<std::string>());
+
+                if ((FLAGS_d.find("MULTI") != std::string::npos) &&
+                    (FLAGS_d.find("CPU") != std::string::npos)) {
+                    // multi-device execution with the CPU + GPU performs best with GPU trottling hint,
+                    // which releases another CPU thread (that is otherwise used by the GPU driver for active polling)
+                    ie.SetConfig({{ CLDNN_CONFIG_KEY(PLUGIN_THROTTLE), "1" }}, "GPU");
+                }
+            } else if (device == "MYRIAD") {
+                ie.SetConfig({{ CONFIG_KEY(LOG_LEVEL), CONFIG_VALUE(LOG_NONE) },
+                              { VPU_CONFIG_KEY(LOG_LEVEL), CONFIG_VALUE(LOG_WARNING) }}, device);
+            }
+        }
+
+        //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~load network to device~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        InferenceEngine::ExecutableNetwork executableNetwork;//
+        executableNetwork = ie.LoadNetwork(network, FLAGS_d);
+
+        //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~try to set optimal number of infer request~~~~~~~~~~~~~~~~~~~~~`
+        if (FLAGS_nireq == 0) {
+            std::string key = METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS);
+            try {
+                FLAGS_nireq = executableNetwork.GetMetric(key).as<unsigned int>();
+            } catch (const details::InferenceEngineException& ex) {
+                THROW_IE_EXCEPTION
+                        << "Every device used with the benchmark_app should "
+                        << "support OPTIMAL_NUMBER_OF_INFER_REQUESTS ExecutableNetwork metric. "
+                        << "Failed to query the metric for the " << FLAGS_d << " with error:" << ex.what();
+            }
+        }
+
+        //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~create inf req~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+        std::vector<InferenceEngine::InferRequest> inferRequests;
```

Comment:
You are already `using namespace InferenceEngine`.
By the way, it seems strange to me to put `using namespace InferenceEngine` but not `std` which is more common. I suggest to use using-directive either for none of the namespaces or for `std` only or for both `std` and `InferenceEngine`. I think the best and the easiest to fix solution is not to place any of using-directive.